src/background.js

```diff
@@ -196,3 +199,20 @@ browser.storage.local.get()
 browser.browserAction.onClicked.addListener(() => {
     browser.sidebarAction.open();
 });
+
+// context menu for 'Send to Notes'
+browser.contextMenus.create({
+  id: 'send-to-notes',
+  title: 'Send to Notes',
+  contexts: ['selection']
+});
+
+browser.contextMenus.onClicked.addListener((info) => {
+  browser.sidebarAction.open();
```

Comment:
@cedricium this doesn't work for me in 58.0.1. How about you?

I get 

> Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist.

<img src=https://user-images.githubusercontent.com/128755/36103329-116e1260-0fdd-11e8-9110-914aa88ede09.gif  width=600 />

Reply:
🤔 it works for me. I introduced the `contextMenus` permission, maybe a fresh install is needed (`npm run build-ck`)?